screens/QrLogin/MyBindedVcs.tsx

```diff
@@ -7,10 +7,16 @@ import {QrLoginRef} from '../../machines/QrLoginMachine';
 import {Icon} from 'react-native-elements';
 import {Modal} from '../../components/ui/Modal';
 import {VcItemContainer} from '../../components/VC/VcItemContainer';
+import {groupBy} from '../../shared/javascript';
 
 export const MyBindedVcs: React.FC<MyBindedVcsProps> = props => {
   const controller = useQrLogin(props);
   const {t} = useTranslation('QrLogin');
+  const [pinned, unpinned] = groupBy(
```

Comment:
Can we move this logic to utliity?
getVCGroupedByPinStatus() => this will return VC grouped by Pin status
Use this at both the places

Reply:
Refactored